modules/codegen/src/smithy4s/codegen/Renderer.scala

```diff
@@ -517,7 +521,7 @@ private[codegen] class Renderer(compilationUnit: CompilationUnit) { self =>
       newline,
       line(s"def to(e: $name) : (String, Int) = (e.value, e.ordinal)"),
       lines(
-        s"val schema: $Schema_[$name] = enumeration(to, valueMap, ordinalMap)",
+        s"val schema: $Schema_[$name] = (enumeration(to, valueMap, ordinalMap): $Schema_[$name]).withHints(hints)",
```

Comment:
Doesn't compile without the type hint here, not sure of another way around this atm